server/src/test/java/com/objectcomputing/checkins/services/questions/QuestionServicesTest.java

```diff
@@ -76,6 +84,25 @@ public void testSaveQuestion_question_not_found() {
 
     }
 
+    @Test
+    public void testUpdate() {
+
+        String fakeQuestionText = "fake question text";
+        Question fakeQuestion = new Question();
+        fakeQuestion.setQuestionid(UUID.fromString(fakeUuid));
+        fakeQuestion.setText(fakeQuestionText);
+        Question updatedFakeQuestion = new Question();
+        updatedFakeQuestion.setQuestionid(UUID.fromString(fakeUuid2));
+        updatedFakeQuestion.setText(fakeQuestionText + "new stuff");
+
+        when(mockQuestionRepository.update(fakeQuestion)).thenReturn(updatedFakeQuestion);
+        when(itemUnderTest.findByQuestionId(fakeQuestion.getQuestionid())).thenReturn(updatedFakeQuestion);
+        Question returned = itemUnderTest.update(fakeQuestion);
+
+        assertEquals(updatedFakeQuestion.getQuestionid(), returned.getQuestionid());
+        assertEquals(updatedFakeQuestion.getText(), returned.getText());
```

Comment:
Adding verify checks at the end would be helpful, e.g. in this case you would want to see questionRepo.update called once.